src/simulation_lifecycle/convert.c

```diff
@@ -11,6 +11,17 @@
 #define PATH_MAX 4096
 #define LENGTH_MAX 200
 #define VALID_FORMAT "[cadmium::celldevs::cell_ports_def<std::string"
```

Comment:
This is compiler-dependant. In MacOS, it has a different format. Additionally, Ubuntu provided yet another different format. I think that we should re-think this assertion, it is too narrow